packages/amplify-category-auth/src/__tests__/commands/import.headless.test.ts

```diff
@@ -0,0 +1,194 @@
+  };
+  const identityPoolDetails = [
+    {
+      IdentityPoolId: IDENTITY_POOL_ID,
+      IdentityPoolName: 'identity-pool',
+      AllowUnauthenticatedIdentities: true,
+      CognitoIdentityProviders: [
+        {
+          ProviderName: `web-provider-${USER_POOL_ID}`,
+          ClientId: WEB_CLIENT_ID,
+        },
+        {
+          ProviderName: `native-provider-${USER_POOL_ID}`,
+          ClientId: NATIVE_CLIENT_ID,
+        },
+      ],
+    },
+  ];
+  const mfaResponse = {
+    SoftwareTokenMfaConfiguration: {
+      Enabled: true,
+    },
+    MfaConfiguration: 'ON',
+  };
+  const getIdentityPoolRolesResponse = {
+    authRoleArn: 'arn:authRole:123',
+    authRoleName: 'authRole',
+    unauthRoleName: 'unAuthRole',
+    unauthRoleArn: 'arn:unAuthRole:123',
+  };
+  // mock fns
+  const cognitoUserPoolServiceMock = jest.fn();
+  const cognitoIdentityPoolServiceMock = jest.fn();
+  const pluginInstanceMock = jest.fn();
+  const getUserPoolDetailsMock = jest.fn();
+  const listUserPoolClientsMock = jest.fn();
+  const getUserPoolMfaConfigMock = jest.fn();
+  const listIdentityPoolDetailsMock = jest.fn();
+  const getIdentityPoolRolesMock = jest.fn();
+  const getProjectConfigMock = jest.fn().mockReturnValue(projectConfig);
+  const getProjectDetailsMock = jest.fn().mockReturnValue(projectDetails);
+
+  beforeAll(() => {
+    const loadResourceParametersMock = jest.fn();
+    const updateAmplifyMetaAfterResourceAddMock = jest.fn();
+    const pluginInstance = {
+      loadResourceParameters: loadResourceParametersMock,
+      createCognitoUserPoolService: cognitoUserPoolServiceMock.mockReturnValue({
+        getUserPoolDetails: getUserPoolDetailsMock.mockResolvedValueOnce(getUserPoolDetails),
+        listUserPoolClients: listUserPoolClientsMock.mockResolvedValueOnce(defaultUserPoolClients),
+        getUserPoolMfaConfig: getUserPoolMfaConfigMock.mockResolvedValue(mfaResponse),
+      }),
+      createIdentityPoolService: cognitoIdentityPoolServiceMock.mockReturnValue({
+        listIdentityPoolDetails: listIdentityPoolDetailsMock.mockResolvedValue(identityPoolDetails),
+        getIdentityPoolRoles: getIdentityPoolRolesMock.mockResolvedValue(getIdentityPoolRolesResponse),
+      }),
+    };
+    mockContext = {
+      amplify: {
+        getProjectConfig: getProjectConfigMock,
+        getProjectDetails: getProjectDetailsMock,
+        updateamplifyMetaAfterResourceAdd: updateAmplifyMetaAfterResourceAddMock,
+        getPluginInstance: pluginInstanceMock.mockReturnValue(pluginInstance),
+        saveEnvResourceParameters: jest.fn(),
+      },
+      print: {
+        warning: jest.fn(),
+        info: (msg: string) => console.log(msg),
+        error: jest.fn(),
+      },
+      parameters: {
+        first: 'mockFirst',
+      },
+      input: {
+        command: 'import',
+      },
+    };
+  });
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('should process command successfully', async () => {
```

Comment:
readability: put empty lines between tests and in between blocks like action and expect blocks.

Reply:
done